apps/src/lib/levelbuilder/lesson-editor/SearchBox.jsx

```diff
@@ -0,0 +1,90 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+import _ from 'lodash';
+
+export default class SearchBox extends Component {
+  static propTypes = {
+    onSearchSelect: PropTypes.func.isRequired,
+    searchUrl: PropTypes.string.isRequired,
+    constructOptions: PropTypes.func.isRequired,
+    additionalQueryParams: PropTypes.object
```

Comment:
is the idea that courseVersionId is required when searching for resources, but may not be required when searching for other things?

Reply:
Yeah, that's the idea. Or if a component wants to also search by a different parameter, that could also be added without having to modify this component.

Reply:
Ya blocks won't need a course version so that seems like a great choice